handler.ts

```diff
@@ -91,6 +91,8 @@ export const makePeliasRequests = async (
   return {
     body: JSON.stringify(mergedResponse),
     headers: {
+      'Access-Control-Allow-Credentials': 'true',
+      'Access-Control-Allow-Origin': '*',
```

Comment:
Do we need also `Access-Control-Allow-Methods`?

Reply:
I've taken this from the [Serverless CORS guide](https://www.serverless.com/blog/cors-api-gateway-survival-guide/) which seems to suggest that `Access-Control-Allow-Methods` is handled by AWS API Gateway

Reply:
Ok, please add a comment in that section of the code (and maybe mention the serverless cors setting) for future reference.